psi4/driver/driver_cbs.py

```diff
@@ -541,7 +553,7 @@ def scf_xtpl_helgaker_3(functionname: str, zLO: int, valueLO: float, zMD: int, v
         Higher zeta level. Should be equal to zLO + 2.
     valueHI
         Higher value used for extrapolation.
-    alpha
+    alpha : float
```

Comment:
Per discussion elsewhere, I thought these kinds of type annotation were no longer wanted.